CMSgov/easi-app
# [EASI-3049] cedar systems team member

# EASI-3049


## Changes and Description

- Create endpoint `myCedarSystems` which derives the requestor's EUA (on the BE) and inserts into parameter list for CEDAR Core API call to `GetSystemSummary`



## How to test this change

- Connect to VPN
- Open Postman (this functionality is not yet available in the UI)
- Import latest Postman collection
- Navigate to `EASI/CEDAR/System Summary/Get My Systems`
- Click `Send`, and you should receive back a number of test systems



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
